src/frontend.md

```diff
@@ -28,7 +28,7 @@ src/
   pages/
     bankAccount/
       BankAccountList/
-        [no nested directories]
+        [no nested directories, but you can add additional nested directory as an exception]
```

Comment:
why?

Reply:
Alexander and Natalie discussed it [here](https://ooosome.slack.com/archives/C024YLUQ6AY/p1663944098427269) and agreed it’s OK as an exception.

Reply:
i only see that they agreed to discuss it in person
if we can add nested directories, it's better to describe what are those directories for
otherwise it can turn into a mess